tests/test_checks.py

```diff
@@ -93,3 +94,22 @@ def test_enum_type_boolean(tmpdir, rc):
                              |         | false
     ''')
     commands.check(context, manifest)
+
+
+def test_filename(tmpdir, rc):
+    context, manifest = load_manifest_and_context(rc, '''
```

Comment:
Here, you should write manifest to a file and only then load it from file. See example here:

https://github.com/atviriduomenys/spinta/blob/master/tests/manifests/test_tabular.py#L9-L10